libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts

```diff
@@ -39,13 +39,13 @@ export const eSettBalanceResponsibleMessages = (apibase: string): BalanceRespons
       receivedDateTime: new Date('2021-02-01T00:00:00.000Z'),
       supplier: '123',
       balanceResponsible: '321',
-      gridArea: '344',
+      gridAreas: '344',
```

Comment:
nitpick: this should probably stay `gridArea`

```suggestion
      gridArea: '344',
```